tasks/steps/rotate_dists.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: 'BLOCK : Rotate distributions'
+  include_tasks: 'blocks/rotate_dists.yml'
+  when:
+    - inventory_hostname in single_instances_for_each_machine
+  tags: cartridge-instances
```

Comment:
Is it correct tag?

Reply:
Yes, why not?

Reply:
If it's a separated step, maybe it's better to remove any tags, but i'm not sure

Reply:
If this is a separate step, it might be better to remove any tags, but I'm not sure